spec/spec_helper.rb

```diff
@@ -32,5 +32,5 @@
 def safe_index_name(name)
   return name if ENV['TRAVIS'].to_s != "true"
   id = ENV['TRAVIS_JOB_NUMBER'].split('.').last
```

Comment:
In the end, the issue wasn't exactly `TRAVIS_JOB_NUMBER` but the fact that we use the last digit. Still, I'd recommend to move to

```ruby
id = ENV['TRAVIS_JOB_NUMBER']
```

Reply:
While it is known to be unique, they're documenting it as internal, so that might not be the safest option in case they ever change the variable name or its behavior.
> TRAVIS_JOB_NUMBER: The number of the current job (for example, "4.1").
> TRAVIS_JOB_ID: The id of the current job that Travis CI uses internally.

Reply:
Why don't we just use the `TRAVIS_JOB_NUMBER` without splitting by the dot?

Reply:
Hmmmm, no idea :) Probably a wrong reason! Feel free to change it.